dist/src/main/dist/simulator-tests/test.properties

```diff
@@ -136,6 +136,9 @@ ReplicatedTTL@class=com.hazelcast.simulator.tests.replicatedmap.ReplicatedMapTim
 ReplicatedTTL@putTTLProb=0.7
 ReplicatedTTL@getProb=0.3
 
+MultiValueMapTest@class=com.hazelcast.simulator.tests.map.MultiValueMapTest
```

Comment:
I'd probably decrease threadcount here to 1 or 2.

`MultiValueMapTest@threadCount=2`